tests/unit/amo/pages/TestLanguageTools.js

```diff
@@ -90,157 +81,129 @@ describe(__filename, () => {
-    expect(dictionary).toHaveLength(1);
-    expect(dictionary.find(Link)).toHaveProp(
-      'children',
-      'le French Dictionary',
+    const langPack = screen.getByClassName(
+      `LanguageTools-in-your-locale-list-item--${ADDON_TYPE_LANG}`,
     );
-    expect(langPack).toHaveLength(1);
-    expect(langPack.find(Link)).toHaveProp('children', 'French Language Pack');
+    expect(langPack).toBeInTheDocument();
+    expect(
+      within(langPack).getByText('French Language Pack'),
+    ).toBeInTheDocument();
   });
 
   it('omits "language tools in your locale" section if none available', () => {
     const { store } = dispatchClientMetadata({ lang: 'pt-BR' });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
+    render({ store, lang: 'pt-BR' });
 
-    expect(root.find('.LanguageTools-in-your-locale')).toHaveLength(0);
+    expect(
+      screen.queryByRole('heading', { name: 'Available for your locale' }),
+    ).not.toBeInTheDocument();
   });
 
   it('renders language packs in the table view for the right language', () => {
-    const { store } = dispatchClientMetadata();
+    const { store } = dispatchClientMetadata({ lang });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
-
-    expect(root.find('.LanguageTools-lang-en-GB')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-en-US')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-fr')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-ur')).toHaveLength(1);
-    expect(root.find('.LanguageTools-lang-zh-TW')).toHaveLength(1);
-
+    render({ store });
+
+    expect(
+      screen.getByClassName('LanguageTools-lang-en-GB'),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByClassName('LanguageTools-lang-en-US'),
+    ).toBeInTheDocument();
+    expect(screen.getByClassName('LanguageTools-lang-fr')).toBeInTheDocument();
+    expect(screen.getByClassName('LanguageTools-lang-ur')).toBeInTheDocument();
+    expect(
+      screen.getByClassName('LanguageTools-lang-zh-TW'),
+    ).toBeInTheDocument();
     // Zulu is not a supported locale on the site but since Firefox UI locales
     // are not necessarily going to match the site locales we should
     // still render the zulu language pack and dictionary.
-    expect(root.find('.LanguageTools-lang-zu')).toHaveLength(1);
+    expect(screen.getByClassName('LanguageTools-lang-zu')).toBeInTheDocument();
   });
 
-  it('renders multiple language tools in a list using LanguageToolList', () => {
-    const { store } = dispatchClientMetadata();
+  it('renders multiple language tools in a list', () => {
+    const { store } = dispatchClientMetadata({ lang });
     store.dispatch(loadLanguageTools({ languageTools }));
 
-    const root = renderShallow({ store });
+    render({ store });
+
+    const dictionaryList = within(
```

Comment:
Just wondering again if we can locate these using something other than `className`? That might not work for the containers, but after we have located the container, could we maybe look for the contents of the containers using strings, rather than `li`? E.g. `expect(within(languagePackList).getByText('something')).toBeInTheDocument()`